pkg/pdutil/pd.go

```diff
@@ -607,3 +608,14 @@ func (p *PdController) Close() {
 	p.pdClient.Close()
 	close(p.schedulerPauseCh)
 }
+
+// FetchPDVersion get pd version
+func FetchPDVersion(ctx context.Context, tls *common.TLS, pdAddr string) (*semver.Version, error) {
+	var rawVersion string
+	err := tls.WithHost(pdAddr).GetJSON(ctx, "/pd/api/v1/config/cluster-version", &rawVersion)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	return semver.NewVersion(rawVersion)
```

Comment:
The raw version string fetch from pd is quoted, I think we need to trim the `"` here.
```
> curl http://172.16.4.5:2379/pd/api/v1/config/cluster-version

"4.0.8"
```